modules/tinymce/src/plugins/save/main/ts/ui/Buttons.ts

```diff
@@ -12,7 +12,7 @@ import * as Options from '../api/Options';
 
 const stateToggle = (editor: Editor) => (api: Toolbar.ToolbarButtonInstanceApi) => {
   const handler = () => {
-    api.setDisabled(Options.enableWhenDirty(editor) && !editor.isDirty());
+    api.setEnabled(!(Options.enableWhenDirty(editor) && !editor.isDirty()));
```

Comment:
```suggestion
    api.setEnabled(!Options.enableWhenDirty(editor) || editor.isDirty()));
```
Maybe? This one I think would read better if we replaced the and unlike the last one 🤷